cloud/worker/main_dacx.go

```diff
@@ -68,5 +68,6 @@ id: how-to-run-a-temporal-cloud-worker-in-go
 title: How to run a Temporal Cloud Worker in Go
 label: Run a Cloud Worker
 description: Use a certificate key pair and your Temporal Cloud Namespace to connect to Temporal Cloud.
+tags: go sdk, code sample
```

Comment:
```suggestion
tags: go sdk, code sample, worker, how-to
```

Do another pass, and try to add any other related tags - features, patterns, doc type etc...